utils/makeappicons.sh

```diff
@@ -139,26 +148,27 @@ map() {
 }
 
 # Now do the actual work
-find application -type f -name "*.svg" | while read -r DST;do
+find $DSTDIR -type f -name "*.svg" | while read -r DST;do
 
     # Find the icon name (base name without extender)
     NAME=$(basename $DST .svg)
 
     # Find the base name of the svg file for this icon
     MAT=$(map $NAME)
     if [[ -z $MAT ]];then
-        echo "Warning: Don't know about $NAME"
+        echo -e "${YELLOW}Warning: No MaterialDesign mapping for ${NAME}${NC}"
         continue
     fi
 
     # So the source file is:
     SRC="$MATERIAL/svg/$MAT.svg"
     if [ ! -f "$SRC" ];then
-        echo "Error: Source for $NAME doesn't exist: $SRC"
-        exit 1
+        echo -e "${RED}Error: Source for ${NAME} doesn't exist: ${SRC}${NC}"
+        continue
     fi
 
     # Replace the icon file with the source file
     cp "$SRC" "$DST" || exit
+    echo "Copied icon for ${NAME}"
```

Comment:
There should be a minification step here (e.g. with svgo).

Reply:
I prefer adding a minify action to `make icons`. In fact, that build target needs to be significantly trimmed since the move to SVG's.

Reply:
Fine by me.